src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -98,4 +110,9 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
     f"DataWord(0x$n%02x)" //would be even better to add a leading zero if odd number of digits
 
   def toBigInt: BigInt = n
+
+  /**
+    * @return Size in bytes excluding the leading 0 bytes
+    */
+  def byteSize: Int = if (isZero) 0 else (n.bitLength - 1) / 8 + 1
```

Comment:
222 is about this: https://github.com/input-output-hk/etc-client/pull/46/files#diff-f692200418f9d3438ebb71f6200c7046R33
but in line above I was talking about this `(n.bitLength - 1) / 8 + 1`
why we subtracting 1 from length?